openwisp_utils/admin_theme/templates/admin/base_site.html

```diff
@@ -18,32 +18,33 @@
 
 {% block menu-bar %}
   <div class="head">
-    <span class="hamburger"><span></span></span>
+    <span tabindex="0" role="button" aria-label="Maxamize menu"class="hamburger"><span></span></span>
```

Comment:
This line has 2 issues:

- "Maxamize menu" should be flagged as translatable
- "class" does not have any preceding whitespace